explorer/src/lib/components/navbar/Navbar.svelte

```diff
@@ -0,0 +1,129 @@
+<script>
+  import { NavList, Select } from "$lib/dusk/components";
+  import "./styles.css";
+
+  /** @type {Number}*/
+  let clientWidth;
+
+  /** @type {HTMLBodyElement}*/
+  let body;
+
+  let hidden = true;
+
+  const navigation = [
+    {
+      link: "/",
+      title: "Chain Info",
+    },
+    {
+      link: "/blocks",
+      title: "Blocks",
+    },
+    {
+      link: "/transactions",
+      title: "Transactions",
+    },
+  ];
+
+  const networks = [
+    {
+      label: "testnet",
+      value: `${import.meta.env.VITE_DUSK_TESTNET_NODE}`,
+    },
+    {
+      label: "devnet",
+      value: `${import.meta.env.VITE_DUSK_DEVNET_NODE}`,
+    },
+  ];
+
+  //@ts-ignore;
+  const bindBody = (node) => {
+    body = node;
+  };
+
+  /**
+   * @param {Boolean} bool
+   */
+  const toggleScroll = (bool) => {
+    if (!body) {
+      return;
+    }
+
+    if (bool) {
+      body.style.overflow = "auto";
+    } else {
+      body.style.overflow = "hidden";
+    }
+  };
+
+  $: toggleScroll(hidden);
+</script>
+
+<svelte:body use:bindBody />
+<svelte:window bind:innerWidth={clientWidth} />
+
+<nav class="dusk-navbar" class:dusk-navbar--menu-hidden={hidden}>
+  <a href="/" class="dusk-navbar__logo">
+    <img src="/dusk_logo.svg" alt="Dusk Logo" />
```

Comment:
Could have been better although we have different logo sizes for desktop and mobile.

Reply:
Yes, I figured that, but I CSS should take precedence when present, right?

My point was give some initial dimensions in the HTML to avoid that shifting.
Also true that probably here will be neglectable, but I hate those shiftings so much.

Reply:
I do not see how we can overcome shiftings as I can set the desktop dimensions as the initial ones but the users loads the page on a mobile screen and vice versa.

Reply:
You can read about it [here](https://web.dev/articles/optimize-cls) and [here](https://blog.logrocket.com/jank-free-page-loading-with-media-aspect-ratios/).
In this case maybe the [sizes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#sizes) attribute is enough?

(another approach for this specific case, which I don't like, would be adding a style tag with media queries inside the svg)

Reply:
(you can explore this in a future issue if you want to merge the PR)